packages/hdwallet-metamask-multichain/src/mipdStore.ts

```diff
@@ -0,0 +1,3 @@
+import { createStore } from "mipd";
+
+export const mipdstore = createStore();
```

Comment:
save a file and just instantiate in the adapter?

Reply:
e8d0d107